0.1.0/lib.typ

```diff
@@ -0,0 +1,159 @@
+#import "@preview/acrostiche:0.3.1": init-acronyms, print-index
+#import "lib/research-questions.typ": init-rqs
+
+// workaround for lack of std scope
+#let std-biblio = bibliography
+
+// store in states such that these need to be passed
+// only once, in the `thesis` function
+#let _authors = state("authors", ())
+#let _title = state("title", [])
+#let _supervisors = state("supervisors", ())
+#let _year = state("year", [2023-2024])
```

Comment:
Might be nice to have the default year be [the current year](https://typst.app/docs/reference/foundations/datetime/)?